### ISSUE TITLE
Gemenviron.profile/exit vs. return

### ISSUE THREAD

**Issue by alexrichert** [Unidata/gempak#47]
Gemenviron.profile has a couple of conditions that result in 'exit'; when the file is sourced (as per compilation instructions), this has the effect of exiting the shell/terminal session entirely, not just the Gemenviron.profile file. Can these be converted to "return" statements so that only the sourcing is cancelled, but the parent shell is not exited?

**Comment by akrherz** [comment]
@alexrichert I agree :), but am unsure if there is a historical legacy as to why it behaves in this manner.  Would you wish to prepare a PR to make this change?
